Mathlib/Analysis/Calculus/ContDiff/Basic.lean

```diff
@@ -71,6 +69,17 @@ theorem iteratedFDeriv_zero_fun {n : ℕ} : (iteratedFDeriv 𝕜 n fun _ : E =>
     rfl
 #align iterated_fderiv_zero_fun iteratedFDeriv_zero_fun
 
+@[simp]
```

Comment:
Could you please cherry-pick changes to this file to a new branch, so that it can be merged before the `prod` part of the PR? If you do this, then please assign the new PR to me (or request a review from me), and I'll merge it quickly.

Reply:
Thanks! Done as #10733